flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaTableITCase.java

```diff
@@ -179,6 +180,66 @@ public void testKafkaSourceSink() throws Exception {
         deleteTestTopic(topic);
     }
 
+    @Test
+    public void testKafkaSourceSinkWithBoundedSpecificOffsets() throws Exception {
```

Comment:
Can we also add a test for timestamps?

Reply:
Sure, I can.

So far I did not want to add too many IT tests, as this PR does not introduce a new feature, but only exposes what has been implemented already for DataStream. I assumed, but I haven't checked, this should've been covered by the DataStream tests.

I left that single test as a smoke test for the entire stack, but I can add one more, which can serve as an example.